### PR TITLE
Conservation form at Manage Projects

### PR DESCRIPTION
**Pull request by rishsingh07**
Added form at Manage projects. 
How to produce:
Sign in, go to projects, click on add conservation projects, see the form.

- [x] Submit for Review button is not yet activated as not mentioned in backend yet.
- [x] image

- [x] In project list, editing the conservation leads to resoration form instead conservation
- [x] Forms in horizontal format from vertical
- [x] chaging fields of the screens as per the purpose of the form.

### CODE REVIEW

**Comment by sagararyal** [comment]
#### 📝 Changed routes:


| [**…d-conservation-project**](https://planet-webapp-5vlbex59u-planetapp.vercel.app/profile/projects/add-conservation-project) | [**…/projects/add-projects**](https://planet-webapp-5vlbex59u-planetapp.vercel.app/profile/projects/add-projects) |
|:-:|:-:|
|<a href="https://planet-webapp-5vlbex59u-planetapp.vercel.app/profile/projects/add-conservation-project">Screenshot of /profile/projects/add-conservation-project</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/lU_4SVvn4_VcfJiob-mz7c_17RhYtspb66iTC4wO5o_s6QzS70RTJsZa2leWJTsjWAMkxIQ2xubu4HT2ScH2yg.png">(view full size)</a> |<a href="https://planet-webapp-5vlbex59u-planetapp.vercel.app/profile/projects/add-projects">Screenshot of /profile/projects/add-projects</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/WEP1MrX0SAeunAVbbw8Qx5VZJ_94IBY1QZSsXneYWYbklOuWB4Gj6B23Cz8jP8ECYn9uht1DJAPyIv1Gfgt3Cg.png">(view full size)</a> |



| [**…ts/restoration-project**](https://planet-webapp-5vlbex59u-planetapp.vercel.app/profile/projects/add-projects/restoration-project) |
|:-:|
|<a href="https://planet-webapp-5vlbex59u-planetapp.vercel.app/profile/projects/add-projects/restoration-project">Screenshot of /profile/projects/add-projects/restoration-project</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/x1-cMtmnroEs6mUlUSaCifAnMKx-auROqoww25izJFDSwjKa1uWYXzI9ahJ7Gyt5dyvB4DtmE1QEXPpBw-QQig.png">(view full size)</a> |







Commit bea05645ec6a8a01245edf5847c5044bbbad0f4a (https://planet-webapp-5vlbex59u-planetapp.vercel.app).

**Comment by norbertschuler** [comment]
- [ ] When moving to the 2nd section on label seems to be missing a translation for `startingProtectionYear`.
- [ ] Initially the date is filled with 2021 and at once shown as an error:

Bildschirmfoto 2021-12-28 um 14 46 40

**Comment by norbertschuler** [comment]
- [ ] If going back to previous sections (from 3rd to 2nd or 1st) a lot of entered information already entered gets lost again and has to be entered again, e.g. the donation amount or ecosystem in the 1st section.

**Comment by norbertschuler** [comment]
- [ ] Many fields in the 2nd section do not show an error message if you forgot to fill in some data. As there is no error the user does not know what mandatory information he has forgotten to enter. Please add error messages for every mandatory field.

**Review by norbertschuler — CHANGES_REQUESTED** [review]
See above comments for issues to fix.

**Comment by norbertschuler** [comment]
- [ ] Testing against the staging backend and trying to edit a project of myself, I am stuck as the API call complains about having too much fields: `Extra fields: classification,visitorAssistance` - not exactly but similar to the issue #1350 :

Bildschirmfoto 2022-01-15 um 10 27 58

File: src/features/user/ManageProjects/conservationComponents/DetailedAnalysis.tsx [hunk 1]
```diff
@@ -0,0 +1,532 @@
...
+            if (siteOwners[i].isSet) {
+                owners.push(siteOwners[i].value)
+            }
+        }
+        const submitData = {
+            projectMeta: {
+                location: data.location,
+                areaProtected: data.areaProtected,
+                employeeCount: data.employeeCount,
+                startingProtectionYear: data.startingProtectionYear.getFullYear(),
```
Comment (norbertschuler) [inline src/features/user/ManageProjects/conservationComponents/DetailedAnalysis.tsx:164]: - [ ] I got a crash here trying to add a new conservation project:

Bildschirmfoto 2022-01-15 um 10 35 10

**Comment by norbertschuler** [comment]
- [ ] I just tried to add a new conservation project with the staging backend and crash instantly at http://localhost:3000/profile/projects/new-project?purpose=conservation (also using the Vercel deployment):

Bildschirmfoto 2022-02-19 um 13 43 53

**Review by norbertschuler — CHANGES_REQUESTED** [review]
This has still issues.

**Comment by norbertschuler** [comment]
This MR unfortunately is in no state able to merge it (bugs and conflicts). Is there anyone who can take over this code or should be abandon it and start over newly?

File: public/static/locales/en/manageProjects.json [hunk 2]
```diff
@@ -123,5 +131,52 @@
...
+  "restorationProject": "Restoration Project",
+  "addProjetDescription": "You're few steps away from showcasing your work on Plant-for-the-planet platform.",
+  "addProjetContact": "If you need any help contact us at ",
+  "areaProtectedInfo": "Add Some Information",
+  "supportLink": "support@plant-for-the-planet.org",
+  "wrongCoordinates": "Wrong Coordinates",
+  "actionInfo": "The actions your organization is taking to conserve the ecosystem.",
+  "mainChallengeInfo": "The biggest challenge this project is facing.",
+  "motivationInfo": "The motivation behind protecting this specific ecosystem.",
+  "hector": "hector",
```
Comment (sagararyal) [inline public/static/locales/en/manageProjects.json:176]: written as `hectare`

File: public/static/locales/en/manageProjects.json [hunk 3]
```diff
@@ -123,5 +131,52 @@
...
+  "areaProtectedValidation": "Please enter Area Protected",
+  "employeeCountValidation": "Please enter Employee Count",
```
Comment (sagararyal) [inline public/static/locales/en/manageProjects.json:178]: these are not necessary as you can make these dynamic. {fieldname} is not valid

Comment (sunilsabatp) [reply public/static/locales/en/manageProjects.json:178]: okay

**Comment by sagararyal** [comment]
#### 📝 Changed routes:


| [**…e/projects/add-project**](https://planet-webapp-mlsfupgs1-planetapp.vercel.app/profile/projects/add-project) | [**…e/projects/new-project**](https://planet-webapp-mlsfupgs1-planetapp.vercel.app/profile/projects/new-project) |
|:-:|:-:|
|<a href="https://planet-webapp-mlsfupgs1-planetapp.vercel.app/profile/projects/add-project">Screenshot of /profile/projects/add-project</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/RCkDEc-jLbYHYg30Qho-vgfPdbgcezP9g714W4tFvxzW2jODqO0Pty9EgeetSo1Y0qUoy4Fj-vgehxiK7hhApg.png">(view full size)</a> |<a href="https://planet-webapp-mlsfupgs1-planetapp.vercel.app/profile/projects/new-project">Screenshot of /profile/projects/new-project</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/762MLE_uYnDWxvlIBT4ctBrN9bd9XZnKYnGrk3rEN0m-gfAMD2ToRye1qjx_oL9K6ZlZJ4G7M_eWn9hDK5TuUg.png">(view full size)</a> |







Commit a3be94d60867bac7abff7ae2ca053e16c7328ecf (https://planet-webapp-mlsfupgs1-planetapp.vercel.app).

**Pull request closed without merging**